src/communication/mpi_global_policy.hpp

```diff
@@ -44,14 +51,6 @@ struct mpi_global_policy {
         return nest::mc::mpi::reduce(value, MPI_SUM);
     }
 
-    template <
```

Comment:
It is dead code: it was replaced with something cleaner a while back and we forgot to remove it at the time.
I took the opportunity to delete it now.